unlock-app/src/components/interface/Header.js

```diff
@@ -54,6 +62,21 @@ export default class Header extends React.PureComponent {
             {title}
           </Title>
         )}
+        <AppButtons>
+          {onAppPage &&
+            appButtons.map(({ Button, page }) => {
+              const isActivePage = page === pathname
+              return (
+                <Button
```

Comment:
can we abstract this behavior into a PageNavButton or something?

Reply:
I don't see why not! Keep it local to `Header` or have it in a separate file?

Reply:
I'd put it with the other buttons, in its own file, since we will probably add other pages later, this sets up a pattern to follow

Reply:
Alright, I'll do that as a separate PR then rebase this one just to keep it from swelling too hugely

Reply:
The buttons were kind of mixed in with the work I had going here... Just crammed it all in here at least for the moment, let me know if I should split.